_posts/2016-05-11-CocoaPods-App-1.0.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "CocoaPods App 1.0"
+author: orta
+categories: cocoapods app release
+---
+
+After about a [year of work](https://github.com/CocoaPods/CocoaPods-app/graphs/contributors), with the release of [CocoaPods 1.0](http://blog.cocoapods.org/CocoaPods-1.0/) the corresponding Mac App is also ready for public usage. 
+
+This is no big reveal, we've been advertising it's existence on our websites for a while. I've already talked a little bit about the tech behind the [Mac App](https://blog.cocoapods.org/CocoaPods-App), so this post is the one that aims to make you say our team should use this instead of the gem.
+
+So let me put [my best hat on](/assets/blog_img/app-1.0/hat-on.jpg), and try to pursuade you that this is a _great idea._
+
+<!-- more -->
+
+## What problem does this solve?
+
+CocoaPods exists within the Ruby ecosystem. The majority of Cocoa developers don't have much overlap with writing ruby programs. Installing and maintaining multiple versions of command-line tools like CocoaPods is something that people have to learn in order to use CocoaPods effectively. 
+
+This creates a problem which [bundler solves](https://guides.cocoapods.org/using/a-gemfile.html) really well, but now you're starting to really dive into the ruby ecosystem and the pre-requisite knowledge to contribute raises again. While I'm on my soap-box, if you're using the cocoapods and fastlane gems in an app, you probably should be using bundler.
+
+So the CocoaPods app hosts everything. Literally, it has it's own self-contained copies of `bzr`, `curl`, `git`, `mercurial`, `ncurses`, `openssl`, `pkg-config`, `ruby`, `scons`, `serf`, `subversion`, `yaml` and `zlib`. This allows the CocoaPods team to ensure consitant behavior across all user setups.
+
+This concept is not new for Cocoa developers, as Xcode has been embedding it's own dependencies inside it's `.app` since it moved to the Mac App Store. The advantage from a user's perspective is that it works, and you can install your own crazy version of git, or curl or ruby, and CocoaPods still works.
```

Comment:
Missing opportunity to say "it just works" 😛 

`curl` is code-formatted above but not here, and ruby isn't capitalized.